package.json

```diff
@@ -77,16 +77,16 @@
     "commander": "^7.2.0",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.15.6",
-    "eslint": "^7.0.0",
+    "eslint": "^7.32.0",
     "eslint-config-airbnb": "^18.2.0",
     "eslint-plugin-import": "^2.25.3",
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-react": "^7.20.6",
-    "eslint-plugin-react-hooks": "^1.7.0",
+    "eslint-plugin-react-hooks": "^4.0.0",
     "husky": "^2.0.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^24.5.0",
-    "jest-cli": "^24.7.1",
+    "jest": "^25.0.0",
+    "jest-cli": "^25.0.0",
```

Comment:
Updated jest version to fix `TypeError: MutationObserver is not a constructor"` when running tests with Node 16.
[Reference](https://github.com/testing-library/dom-testing-library/issues/477#issuecomment-617652033)